src/app/zap-templates/zcl/zcl-with-test-extensions.json

```diff
@@ -745,7 +745,6 @@
             "AllocatedVideoStreams",
             "AllocatedAudioStreams",
             "AllocatedSnapshotStreams",
-            "RankedVideoStreamPrioritiesList",
```

Comment:
`StreamUsagePriorities` needs to be added back.

Reply:
Why?  It's a list, right?  Lists don't need to go in here at all.  In fact, you could remove the other list bits as well....

Reply:
This should only contain Structs, yeah?  I was wondering why all the attributes were listed!
Fixed!

Reply:
This _has_ to contain attributes of struct type, _may_ contain any attributes implemented via CommandHandlerInterface, and _should_ contain any non-list-type attributes implemented via CommandHandlerInterface.

Yes, I know, it's a somewhat complicated condition.  :(